axelor-bank-payment/src/main/resources/views/BankReconciliationLine.xml

```diff
@@ -6,9 +6,9 @@
   <grid name="bank-reconciliation-line-grid" title="Bank reconciliation Lines"
     model="com.axelor.apps.bankpayment.db.BankReconciliationLine" canDelete="false" editable="false"
     canNew="false">
-    <button onClick="action-bank-reconciliation-line-method-set-selected" name="select"
+    <button onClick="save,action-bank-reconciliation-line-method-set-selected" name="select"
       title="Select / Unselect" icon="fa-check-circle" showIf="!isSelectedBankReconciliation"/>
-    <button onClick="action-bank-reconciliation-line-method-set-selected" name="select"
+    <button onClick="save,action-bank-reconciliation-line-method-set-selected" name="select"
```

Comment:
Why a save is introduce here ?

Reply:
because the correct button does not save the reconciliation as the comments field is mandatory. So when clicking on correct and then clicking on select, the reconciliation return into validated status